docs/index.md

```diff
@@ -112,9 +112,9 @@ reading the plugins README, or as a last resort, reading their setup.py.
 
 Initial installation:
 
-1. Observe the latest branch of `pulp_installer`, and what version of pulpcore it installs (3.6.4).
+1. Observe the latest branch of `pulp_installer`, and what version of pulpcore it installs (3.6.5.post2).
 1. Confirm that all the latest stable releases of your desired plugins are compatible with pulpcore
-   3.6.4, such as by reading the release announcement email thread for pulpcore 3.6.4, reading the
+   3.6.5.post2, such as by reading the release announcement email thread for pulpcore 3.6.5.post2, reading the
```

Comment:
Few lines bellow need to be updated:
s/3.3.z/3.5.z/
s/3.2.z/3.4.z/

No one was updating it in the past, they were forgotten until 3.8 release or so.